Solutions/Votiro/Analytic Rules/VotiroFileBlockedFromConnector.yaml

```diff
@@ -0,0 +1,42 @@
+id: 17bf3780-ae0d-4cd9-a884-5df8b687f3f5
+name: Votiro - File Blocked from Connector
+description: |
+ 'The analytic rule is intended to detect when a file is blocked by Votiro Sanitization Engine due to a specific policy, and notify the appropriate parties so that they can take appropriate action. The alert message will state that a file with a specific name and hash value was blocked by Votiro Sanatization Engine due to a specific policy name, and that more details can be found at a specific incident URL.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: Votiro
+    dataTypes:
+     - CommonSecurityLog 
+queryFrequency: 10m
+queryPeriod: 10m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - DefenseEvasion
+  - Discovery
+  - Impact
+relevantTechniques:
+  - T1036
+  - T1083
+  - T1057
+  - T1082
+  - T1565
+  - T1498
+  - T0837
+query: let Votiro_view  = view () {
+  VotiroEvents
+   | parse-where SanitizationResult with "Blocked" blocked
+   | where PasswordProtected == "false" and From =~ "null"
```

Comment:
Use =~ instead of ==

Reply:
Hi @devikamehra,
We tried =~ as you suggested but while testing we faced an error showing:
![image (1)](https://user-images.githubusercontent.com/111038486/227932383-550e1e38-8b33-4278-b209-6e85955ba634.png)
So we are not making any changes to this request.
Thanks,
Sambhrant.